src/test/scala/com/github/sstone/amqp/ChannelSpec.scala

```diff
@@ -13,11 +13,11 @@ import com.rabbitmq.client.ConnectionFactory
 import com.github.sstone.amqp.Amqp._
 import scala.util.Random
 
-class ChannelSpec extends TestKit(ActorSystem("TestSystem")) with WordSpecLike with ShouldMatchers with BeforeAndAfter with ImplicitSender {
+trait ChannelSpecNoTestKit extends WordSpecLike with ShouldMatchers with BeforeAndAfter {

```

Comment:
The tests didn't pass for me. Specifically, I had issues with `ConsumerSpec`. The test suite extends `TestKit`, and then uses `TestProbe`... but that actually reuses ***the same actor across all tests in the suite***. The tests mostly worked when run individually, but would run into issues when run all together via `sbt test`. 

This change lets each test create its own actor. See also https://hseeberger.gitlab.io/2017/09/13/how-to-use-akka-testkit.html